packages/datastore/cli/src/generate-documents/createJsonSchema.ts

```diff
@@ -24,15 +21,33 @@ const getFieldParameters = (fieldName: string, type: GraphQLOutputType): any =>
 
 const getModelProperties = (model: ModelDefinition, primaryKey: string) => {
   const fieldMap = model.graphqlType.getFields();
+  const keys = Object.keys(fieldMap);
+
+  const relationships = model.relationships
```

Comment:
This is amazing. We do not need to use delta type and getting all the information.